recipes/google-re2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "google-re2" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-re2-{{ version }}.tar.gz
+  sha256: 21c8adc296360de1ff426baa38c712eada622c2858d195eb487e415d94194e91
+
+build:
+  skip: true  # [win]
```

Comment:
The windows build failed in ways that suggested to me that Windows may not be supported.  If someone is interested in Windows support, we can try to figure it out in the feedstock.  I don't need the Windows build for Apache Airflow, since it doesn't support Windows either.